torch/quantization/fake_quantize.py

```diff
@@ -43,3 +44,16 @@ def forward(self, X):
                 X, self.scale.double(), self.zero_point.long(), self.quant_min,
                 self.quant_max)
         return X
+
```

Comment:
Comment on line 39: Can we remove if self.training: just self.enable_fq should be sufficient. This allows us to use fake quant in eval to model quantization errors.

Reply:
Sure